packages/tocco-ui/src/Panel/PanelBody.js

```diff
@@ -9,7 +9,13 @@ import StyledPanelBody from './StyledPanelBody'
 class PanelBody extends React.Component {
   state = {}
 
-  setHeight = () => {
+  setHeight = a => {
```

Comment:
please remove this change. i will create a separate task to solve the performance issues.